src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -800,10 +797,12 @@ function bindCopyEvents() {
     });
 }
 
-function getQuestionIdSuffix(questionNum) {
-    var isValidQuestionNumber = questionNum > 0 || questionNum === NEW_QUESTION;
-    
-    var idSuffix = isValidQuestionNumber ? '-' + questionNum : '';
-    return idSuffix;
+function getQuestionId($elementInQuestionForm) {
+    var $questionForm = $elementInQuestionForm.closest('form');
+    var id = $questionForm.attr('id');
+    if (id.endsWith('-' + NEW_QUESTION)) {
+        return NEW_QUESTION;
+    }
+    return id.split('-').slice(-1)[0];
```

Comment:
Not sure if this is readable enough, would saving it as an array first then accessing it eg. `arr[arr.length - 1]` be better here?